README.md

```diff
@@ -19,28 +19,28 @@ build dashboards or use Explore with X-Ray to look at traces, analytics, or insi
 | Auth Provider            | Specify the provider to get credentials.                                                                                |
 | Credentials profile name | Specify the name of the profile to use (if you use `~/.aws/credentials` file), leave blank for default.                 |
 | Assume Role Arn          | Specify the ARN of the role to assume.                                                                                  |
-| External ID              | If you are assuming a role in another account, that has been created with an external ID, specify the external ID here. |
+| External ID              | If you are assuming a role in another account that was created with an external ID, specify the external ID here. |
 
 ## Authentication
 
-In this section we will go through the different type of authentication you can use for X-Ray data source.
+In this section we will go through the different type of authentication you can use for the AWS X-Ray data source.
 
 ### IAM Roles
 
-Currently all access to X-Ray is done server side by the Grafana backend using the official AWS SDK. If your Grafana
+Currently all access to X-Ray is done server-side by the Grafana backend using the official AWS SDK. If your Grafana
 server is running on AWS you can use IAM Roles and authentication will be handled automatically.
 
 See the AWS documentation on [IAM Roles](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html)
 
-> **Note:** [AWS Role Switching](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-cli.html) is not supported at the moment.
+> **Note:** [AWS Role Switching](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-cli.html) is not currently supported.
```

Comment:
@lwandz13 , I'm generally in favor of not including adverbs in tech docs as they are not always necessary. Up to you, just a nit.